### PR TITLE
OCLOMRS-360: Add more content about the concept in the preview modal

### PR DESCRIPTION
**Pull request by Tittoh**
# JIRA TICKET NAME:
[Add more content about the concept in the preview modal](https://issues.openmrs.org/browse/OCLOMRS-360)

# Summary:
On this ticket, it’s generally important that the “Preview Concept” screen shows more information. Right now it actually shows less than is shown in the table, because it doesn’t show the Class and Datatype).

### CODE REVIEW

File: src/components/bulkConcepts/component/PreviewCard.jsx [hunk 1]
```diff
@@ -4,24 +4,69 @@ import CardBody from './CardBody';
...
+    id,
+    url,
+    version,
+    created_on,
+    type,
+    concept_class,
+    datatype,
+    source,
+    owner,
+    owner_type,
```
Comment (AGMETEOR) [inline src/components/bulkConcepts/component/PreviewCard.jsx]: @Tittoh did linting come into play here?

Comment (Duncanian) [reply src/components/bulkConcepts/component/PreviewCard.jsx]: @AGMETEOR , I think it had to because he also had to add more props there.

**Comment by dkayiwa** [comment]
@Tittoh do you know why this ticket is still not yet merged?

**Comment by Tittoh** [comment]
@dkayiwa there was there was feedback from Darius that I was still working on.

**Comment by dkayiwa** [comment]
@Tittoh how far are you with working on it?

**Comment by Tittoh** [comment]
I am resolving the merge conflicts and removing console warnings.

**Pull request merged**